### PR TITLE
Case contacts contact made default yes

### LINKED ISSUE #6016: Make "a. Was contact made?" defaulted to "Yes" on Case Contact Form
## What type(s) of user does this feature affect?
 - volunteers
 - supervisors
 - admins

## Description
Please make "a. Was contact made?" defaulted to "Yes"

## Screenshots of current behavior, if any
image

## How to access the Case Contact Form
- Go to "Case Contacts" on left nav bar
- Click "All"
- Click "+ New Case Contact" button

## How to access the QA site
_Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin
- all_casa_admin1@example.com view site as an all casa admin
  - go to `/all_casa_admins/sign_in`  

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in [slack](https://join.slack.com/t/rubyforgood/shared_invite/zt-21pyz2ab8-H6JgQfGGI0Ab6MfNOZRIQA) #casa channel to ask questions quickly. And [discord](https://discord.gg/qJcw2RZH8Q) for office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by panacotar**
### What github issue is this PR for, if any?
Resolves #6016

### What changed, and _why_?
This changes the default value for the `contact_made` column to `true`. This was done via a migration. Please let me know if that's alright or if we should do it via helpers and change the [radio_button checked value](https://github.com/rubyforgood/casa/blob/ac35c957d4a84445fba0c20a1819cdc16e92a247/app/views/case_contacts/form/details.html.erb#L35).

I'm still learning and I am very much open to suggestions. 

### How is this **tested**? (please write tests!) 💖💪  

One of the specs was testing if `@case_contact.contact_made` was blank/false. With this change, the `case_contact` column will be true so the test failed. I removed this attribute from the tests as with the new migration, it is not the case for it to be blank. 

### Screenshots please :)
_Run your local server and take a screenshot of your work! Try to include the URL of the page as well as the contents of the page._ 

### Feelings gif (optional)
_What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:_
`alt text`

alt text

### CODE REVIEW

**Comment by elasticspoon** [comment]
I think I would rather we do the change via that helper method that you pointed out. 

The logic being, typically the data base schema essentially outlines the "axioms" or fundamental truth of the application (ex: a user can only belong to one organization, case contact must belong to a case).

In this case I would consider the default to be more of "business" logic. This is what the users want at the moment and they might just want it to change again in the future, we have no reason to tie ourselves to that decision. Thus, its better to do it in the ruby code and that helper looks like a great place for it.

**Comment by DariusPirvulescu** [comment]
Thanks for the help.
What if we create a CaseContact instance and set the `contact_made: true`?
This will remove the need for the `set_contact_made_false` helper and its tests.
Previously, the `set_contact_made_false` helper checked if `case_contact.persisted?`, but it always persisted as we created the instance beforehand.

Please let me know what you think of this solution.

File: app/views/case_contacts/form/details.html.erb [hunk 1]
```diff
@@ -24,15 +24,14 @@
         <h5 classs="mb-3"><label>a. Was contact made?</label></h5>
         <div class="form-check radio-style mb-20">
           <%= form.radio_button :contact_made, true,
-            checked: @case_contact.contact_made,
+            checked: true,
```
Comment (elasticspoon) [inline app/views/case_contacts/form/details.html.erb:27]: I don't think we need this line if we are setting `contact_made: true` in the controller.

**Review by elasticspoon — APPROVED** [review]
Small bit of feedback but overall LGTM! Thanks!

File: app/views/case_contacts/form/details.html.erb (hunk 1 above)
Comment (DariusPirvulescu) [reply app/views/case_contacts/form/details.html.erb]: That is true, thanks for catching it!

**Pull request merged**